nuxeo-importer-queues/src/main/java/org/nuxeo/ecm/platform/importer/queue/manager/CQManager.java

```diff
@@ -78,8 +78,22 @@ public void init() {
                 tailers.add(queue.createTailer().toEnd());
             }
         }
+        isSetup = true;
     }
 
+    @Override
+    public void stop() {
+        try {
+            queues.stream().forEach(ChronicleQueue::close);
```

Comment:
You can skip the `stream()` part and directly call `.forEach` on any `Iterable`.